planning/costmap_generator/config/costmap_generator.param.yaml

```diff
@@ -0,0 +1,22 @@
+/**:
```

Comment:
@satoshi-ota 
is it ok to add planning/costmap_generator/config/costmap_generator.param.yaml in uverse?